Clover/app/src/main/java/org/floens/chan/core/database/DatabaseHelper.java

```diff
@@ -256,6 +259,36 @@ public void onUpgrade(SQLiteDatabase database, ConnectionSource connectionSource
                 Logger.e(TAG, "Error upgrading to version 25", e);
             }
         }
+
+        if (oldVersion < 26) {
+            try {
+                postHideDao.executeRawNoArgs("ALTER TABLE threadhide RENAME TO " + POST_HIDE_TABLE_NAME + ";");
```

Comment:
Woah, this is actually bad. Database migrations should not have variables in them. I anyone renames at least one of them everything will break. Should replace them all.